### DISCUSSION TITLE
Imported .db not recognized

### DISCUSSION THREAD

**Discussion by blogdemoi** [mcguirepr89/BirdNET-Pi#658] [Q&A] [answered]
Hello,

I had problems with my Birdnet-Pi not recognizing sounds anymore. So I backed-up the .db and erased all data from the "tools" section. Then rebooted and the Pi started again to record.

I then stopped all active services, deleted the new .db and imported the .db I previously saved and chmoed it. The Pi records and recognises successfully new birds but the past detections (from the backed up .db) don't show in the "Recordings" page. In the "By date" section, no detections (before the back up) appear and in the "By species" they do appear (but not the audio recordings as I deleted them).

I checked the .db from "tools/Database Maintenance" and the detections are listed fine there, going back to 2022-08-30.

What can be wrong?

Thank you.

**Comment by ehpersonal38** [comment, marked as answer]
The "Recordings" page will only show detections that actually have audio files present. If you go to "Species Stats" I suspect you'll be able to see all your detections as expected.

**Reply by blogdemoi** [reply]
Ah, yes, you are right, they do appear in "Species Stats", thank you.
